api/v1alpha1/atlasmigration_types.go

```diff
@@ -68,6 +68,8 @@ type (
 		Dir Dir `json:"dir"`
 		// RevisionsSchema defines the schema that revisions table resides in
 		RevisionsSchema string `json:"revisionsSchema,omitempty"`
+		// BaselineVersion defines the baseline version of the database on the first migration.
+		BaselineVersion string `json:"baselineVersion,omitempty"`
```

Comment:
```suggestion
		Baseline string `json:"baseline,omitempty"`
```
I prefer to use `baseline` as it's more closely related to the Atlas CLI